bot/cogs/grades_data/grades_cog.py

```diff
@@ -30,39 +30,23 @@ class gradesCog(commands.Cog):
                     '2019Fall.csv', 
                     '2020Spring.csv']
 
+    MIN_YEAR = 2014
+
     def __init__(self, bot):
         self.bot = bot
         self.fileLines = []
 
         self.master_list = {}
         self.master_prof_list = {}
+        self.MIN_YEAR = 2014
```

Comment:
you dont need this, this is a separate instance const, we can use just the module const one